README.md

```diff
@@ -1,2 +1,42 @@
-# setup-atlas
-A GitHub Action to install ariga/atlas in your workflows.
+# setup-action
+
+A GitHub Action for installing up [Atlas](https://github.com/ariga/atlas).
+
+This action is used for installing Atlas on the Github runner.
+
+> Please note that for most use cases, you should first consider using [Atlas Action](https://github.com/ariga/atlas-action) which can save you a lot of work.
+
+### Usage
+
+Add `.github/workflows/setup-atlas.yaml` to your repo with the following contents:
```

Comment:
Since state isn't shared between workflows, this action is only useful as a first step in a workflow that uses Atlas, so `setup-atlas.yaml` probably will not be its name. You can show something like `deploy-schema.yaml` that installs atlas and then runs `atlas schema apply --env prod` or similar